packages/serialize/src/index.js

```diff
@@ -89,7 +89,7 @@ if (process.env.NODE_ENV !== 'production') {
     }
 
     if (
-      key.charCodeAt(1) !== 45 &&
+      (key.charCodeAt(0) !== 64 /* @ */ || key.charCodeAt(1) !== 45) /* - */ &&
```

Comment:
im not sure what case is covered by this part `key.charCodeAt(1) !== 45) /* - */` as there is no test for this, is it supposed to handle css variables?

Reply:
I believe it is.